pkg/webhooks/errors.go

```diff
@@ -37,9 +39,44 @@ func IsAdmissionCantConnect(err error) bool {
 	return false
 }
 
+func IsCantConnectAPIServer(err error) bool {
```

Comment:
The `IsCant` part of the function name seems a bit strange, but since we are following the naming convention as the other functions in this package so I guess it's fine to have it like this

Reply:
The other "cantConnect" message (that I didn't touch) is `cantConnectErrorMsg` and for that the function name is `IsAdmissionCantConnect`.

I don't necessarily like the `IsCant...` particle but : 
* not sure that `...Admission...` is all that accurate in the context so I avoided copying that part (correct me if I'm wrong!)
* in the k8s repos we often see function names for error comparison like `IsNotFound` which is not really different to this in my opinion so, following that logic, it doesn't feel like a code smell to have that name

Reply:
I don't really have a good suggestion either, so I am fine with leaving it like this